tests/Microsoft.Identity.Test.Unit.net45/PublicApiTests/ConfidentialClientApplicationTests.cs

```diff
@@ -783,6 +783,26 @@ public async Task AuthorizationCodeRequestTestAsync()
                 }).ConfigureAwait(false);
         }
 
+        [TestMethod]
+        [TestCategory("ConfidentialClientApplicationTests")]
```

Comment:
I don't think we need this category.

Reply:
?

---
In reply to: [246008037](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/759#discussion_r246008037) [](ancestors = 246008037)